docker/WebSphere/src/configuration/02-configureJdbc.py

```diff
@@ -67,7 +67,7 @@ def createDatasource(datasourceName, providerName, authAlias, properties):
 	])
 
 createDatasource('ibis4test-mariadb', 'MariaDB JDBC Driver', authAliasName, [
-		[['name', 'Url'], ['value', 'jdbc:mariadb://${jdbc.hostname}:3306/testiaf']],
+		[['name', 'url'], ['value', 'jdbc:mariadb://${jdbc.hostname}:3306/testiaf']],
```

Comment:
Moet dit niet ook voor Wildfly / JBoss?

Reply:
Nope die gaan goed ? 🍓 
De driver method is setUrl, blijkbaar wilt WebSphere echt alles lowercase?

Reply:
Raar, 'url' zou dan de verwachtte spelling zijn voor dat property.